jobs/future-effective-filings/requirements/prod.txt

```diff
@@ -7,5 +7,9 @@ flask-jwt-oidc>=0.1.5
 python-dotenv
 requests
 sentry-sdk[flask]
-
+python-dateutil
+asyncio-nats-client
+asyncio-nats-streaming
 git+https://github.com/bcgov/lear.git#egg=registry_schemas&subdirectory=schemas
+git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
```

Comment:
I'd remove legal_api from requirements since it's not actually used; otherwise confusing. Also I don't think registry_schema is used? Definitely entity_queue_common is.